Marlin/src/gcode/sdcard/M20-M30_M32-M34_M928.cpp

```diff
@@ -115,6 +115,14 @@ void GcodeSuite::M27() {
       command_queue_port[cmd_queue_index_r]
     #endif
   );
+  #if ENABLED(AUTO_REPORT_SD_STATUS)
+  if (parser.seenval('S'))
+    card.set_auto_report_interval(parser.value_byte()  
+		#if NUM_SERIAL > 1
```

Comment:
Watch out for tabs.

Reply:
ouch ! Yes, sorry about that. Should I correct them ? Or will you ?

Reply:
Got 'em! Needed a rebase anyway.